### ISSUE TITLE
"/funnyguilds funnybin" giving error: Server returned HTTP response code: 502

### ISSUE THREAD

**Issue by Arturro43** [FunnyGuilds/FunnyGuilds#1433]
So, telemetry is broken, I guess...

Full error report:

[here](https://pastebin.com/raw/QZmBZkcV)

**Comment by bopke** [comment]
Howdy,

I wanna thank you for reporting that pesky bug in our software. You're a real sharpshooter for helping us fix it up.

After takin' a good look at it, we found out that the darn feature has been busted for over two years now. Ain't that a kick in the pants? We know it's been a real headache for y'all, and we're sorry for that.

But to be honest, that feature ain't been bringin' no value to our project. Fixin' it up would require a whole heap of resources and time, and we reckon it just ain't worth it. That's why we've decided to ride it out and remove it from our software to keep things streamlined.

We sure do appreciate your understanding and support, partner. If y'all have any more questions or concerns, just holler at us.

Thanks for stickin' with us and keepin' the software ridin' smooth.

Best regards,
Bopke
